### PR TITLE
website/blog: draft for happy bday blog

### PR DESCRIPTION
**Pull request by tanberry**
Very drafty:

-- still need to get image in metadata to work
-- still need to actually read again when i am not exhausted
-- Jens and all please carefully review all of it, and especially the What's next section.

### CODE REVIEW

File: website/blog/2023-11-1-happy-birthday-to-us/item.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### A founder’s brain and heart
+
+Our team at authentik has a shared love of building things, and that shapes both how we work together and also our product, even how we communicate with our community.
+
+An interesting offset to our shared love of building is the shared sense of humility, of which we get daily doses from Jens.
+
+> To build boldly yet with humility is what sets some founders apart from others.
+
+The tone and espirit of the company is one reason it’s so meaningful to celebrate our 1-year birthday; we can happily celebrate a hard year of doing things with full, enthusiastic engagement. At authentik, nerdiness is embraced, technical curiousity flourishs, and transparency is a big part of our nature. Speaking of how we communicate with our community, our Discord forum is (in addition to GitHub) an important place where transparency matters. For example, we recently asked our community what they preferred for a release cycle. Based on the answers, we lengthened the release time from from monthly to every two or three months.
```
Comment (kensternberg-authentik) [inline website/blog/2023-11-1-happy-birthday-to-us/item.md:92]: Typo: "flourishs" -> "flourishes"

File: website/blog/2023-11-1-happy-birthday-to-us/item.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Moving from a role of solo creator of an open source project, to being primary maintainer of a popular, growing project, to suddenly being CTO of a company based on that project is a quite a transition. A natural question we wanted to ask Jens is “What’s been the hardest thing about building a company?” His answers:
+
+-   “Recognizing and accepting that you don’t get to work on only what you want to, 100% of time… “
+-   “Learning to delegate, learning to let go a bit, trusting others to do it in their way, in the right spirit. Especially letting others get into the code… I’ve learned that instead of saying ‘I would not have done it this way’, I instead measure the success of the change itself.”
+
+### What’s up next?
+
+Going forward, we want to of course keep our focus on building features and supporting authentication protocols that our users want, but we have also identified several specific goals for this coming year:
```
Comment (kensternberg-authentik) [inline website/blog/2023-11-1-happy-birthday-to-us/item.md:101]: Suggestion: The "... of course ..." kinda breaks the flow.  It works verbally, but read silently it feels a bit like a hiccup.

**Review by kensternberg-authentik — COMMENTED** [review]
Just some generic grammar things.

File: website/blog/2023-11-1-happy-birthday-to-us/item.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+> How does this thing work, from whom can I learn more, and with whom can I share my knowledge?
+
+We have that team now, and are grateful for it. Celebrating the one-year mark of Authentik Security means a lot to us!
+
+**Keep those PRs merging**
+
+Keeping new functionality rolling out (and keeping up with Issues and PRs in our repository) never slowed down much, even during the period of incorporating as a company and building a team. Support for new providers, becoming [OpenID certified](https://goauthentik.io/blog/2023-03-07-becoming-openid-certified-why-standards-matter), adding support for SCIM and RADIUS protocols, and a [lot more](https://goauthentik.io/docs/releases).
+
+Right at the end of our first year, we released our [Enterprise version](https://goauthentik.io/blog/2023-08-31-announcing-the-authentik-enterprise-release), with dedicated support. And just last week, we rolled out one of the most important capabilities in an identity management platform; [RBAC](https://goauthentik.io/docs/user-group-role/access-control/) (role-based access control).
```
Comment (rissson) [inline website/blog/2023-11-1-happy-birthday-to-us/item.md:70]:
```suggestion
Right at the end of our first year, we released our [Enterprise version](https://goauthentik.io/blog/2023-08-31-announcing-the-authentik-enterprise-release), with dedicated support. And just last week, we rolled out one of the most important capabilities in an identity management platform: [RBAC](https://goauthentik.io/docs/user-group-role/access-control/) (role-based access control).
```

File: website/blog/2023-11-1-happy-birthday-to-us/item.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+N**ew processes, new ideas, and expected growing pains**
+
+With a new team, come new processes. Someone has to decide which emoji to use for which infrastructure task that’s completed.
```
Comment (rissson) [inline website/blog/2023-11-1-happy-birthday-to-us/item.md:74]: Did we?

File: website/blog/2023-11-1-happy-birthday-to-us/item.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+-   Research and implement new functionality around remote machine access and management
+-   Defining increasingly robust tests and checks for our CI/CD pipeline and build process
+-   Implementing even stronger integration and migration testing, both automated and manual
+-   Spending more time on outreach and learning from our users about what you all want and where we can improve.
+
+This space of security and authentication is a hard space, especially with larger configurations with multiple providers, large user sets to be imported, and the absolute minute-by-minute race against malevolent hackers.
+
+Oh, and then there is that business of actually promoting and selling your product. But, as a team, we are proud of the product and excited to share it with others who need a solid, secure authentication platform.
+
+Thanks for joining us on this celebration of our one-year birthday, and let us know any thoughts you might have. You can send an emial to hello@authentik.io, or find us on GitHub or Discord.
```
Comment (rissson) [inline website/blog/2023-11-1-happy-birthday-to-us/item.md:113]:
```suggestion
Thanks for joining us on this celebration of our one-year birthday, and let us know any thoughts you might have. You can send an email to hello@authentik.io, or find us on GitHub or Discord.
```

**Review by rissson — DISMISSED** [review]

File: website/blog/2023-11-1-happy-birthday-to-us/item.md (hunk 3 above)
Comment (tanberry) [reply website/blog/2023-11-1-happy-birthday-to-us/item.md:70]: Wow, great catch, thank you.

**Review by kensternberg-authentik — APPROVED** [review]
Two [`polish`](https://conventionalcomments.org/#:~:text=Polish%20comments%20are%20like%20a%20suggestion%2C%20where%20there%20is%20nothing%20necessarily%20wrong%20with%20the%20relevant%20content%2C%20there%27s%20just%20some%20ways%20to%20immediately%20improve%20the%20quality.) level comments, but otherwise great.

**Pull request merged**